package.json

```diff
@@ -17,25 +17,25 @@
     "@bcrs-shared-components/approval-type": "1.0.19",
     "@bcrs-shared-components/base-address": "2.0.3",
     "@bcrs-shared-components/breadcrumb": "2.1.15",
-    "@bcrs-shared-components/business-lookup": "1.3.1",
+    "@bcrs-shared-components/business-lookup": "1.3.3",
     "@bcrs-shared-components/certify": "2.1.15",
     "@bcrs-shared-components/completing-party": "2.1.30",
     "@bcrs-shared-components/confirm-dialog": "1.2.1",
     "@bcrs-shared-components/contact-info": "1.2.15",
     "@bcrs-shared-components/corp-type-module": "1.0.16",
     "@bcrs-shared-components/correct-name": "1.0.46",
     "@bcrs-shared-components/court-order-poa": "3.0.11",
-    "@bcrs-shared-components/date-picker": "1.2.15",
+    "@bcrs-shared-components/date-picker": "1.2.55",
```

Comment:
This big of a jump shouldn't prevent the date picker from not working in other filings right? I really hope not.

Reply:
A large part of this jump is due to shared interface updates. I did also previously fix the "disabled" functionality of the date picker, and changed it to always honour the input rules.

I will check other filings. Should be OK.

Reply:
Sample firm registration for testing Start Date:
https://business-create-dev--pr-691-nozaqqyw.web.app//registration-define-business?id=TS2z8VCFiJ

Sample coop dissolution for testing Resolution Date:
https://business-create-dev--pr-691-nozaqqyw.web.app//dissolution-resolution?id=CP1002592

Sample firm dissolution for testing Dissolution Date:
https://business-create-dev--pr-691-nozaqqyw.web.app//dissolution-firm?id=FM1009715

The Date Picker seems to work fine in the above filings.

Reply:
Awesome, thank you so much! Resolving.